lib/services/cordova-migration-service.ts

```diff
@@ -352,5 +355,19 @@ export class CordovaMigrationService implements ICordovaMigrationService {
 			return plugin;
 		});
 	}
+
+	private migrateWebView(oldFrameworkVersion: string, newFrameworkVersion: string): IFuture<void> {
+		return (() => {
+			// For Cordova versions below 5.0.0 with WKWebView we need to set the WKWebView to com.telerik.plugins.wkwebview.
+			// For Cordova version 5.0.0 and above with WKWebView we need to set the WKWebView to cordova-plugin-wkwebview-engine.
+			let currentWebViewName = this.$webViewService.getCurrentWebViewName(this.$projectConstants.IOS_PLATFORM_NAME);
+			let currentWebView = this.$webViewService.getWebView(this.$projectConstants.IOS_PLATFORM_NAME, currentWebViewName, oldFrameworkVersion);
```

Comment:
You'll execute a lot of operations even in case the currentWebView is the same as the new one (for example migration from 3.7.0 to 4.0.0). Can we optimize this and do nothing in such cases?